content/actions/creating-actions/setting-exit-codes-for-actions.md

```diff
@@ -40,7 +40,7 @@ For more information, see "[Creating a JavaScript action](/articles/creating-a-j
 
 If you are creating a Docker container action, you can set a failure exit code in your `entrypoint.sh` script. For example:
 
-```
+```shell
```

Comment:
This code fence actually removes `<condition>` in the rendered output. You'll need to add `{% raw %}` and `{% endraw %}` around the code fence. 

https://docs-772--code-fence-languages.herokuapp.com/en/free-pro-team@latest/actions/creating-actions/setting-exit-codes-for-actions#setting-a-failure-exit-code-in-a-docker-container-action